src/shared/yagna/repository/debit-note-repository.ts

```diff
@@ -41,7 +40,7 @@ export class DebitNoteRepository implements IDebitNoteRepository {
     const providerInfo = {
       id: model.issuerId,
       walletAddress: model.payeeAddr,
-      name: (agreement.offer.properties as ProposalProperties)["golem.node.id.name"],
+      name: agreement.offer.properties["golem.node.id.name"] ?? "",
```

Comment:
Note for reviewers: That's after all proposal properties were marked as optional when I switched to `Partial`